### DISCUSSION TITLE
Audio / Ultrasonic field recorders

### DISCUSSION THREAD

**Discussion by DD4WH** [mcguirepr89/BirdNET-Pi#110] [General]
This thread is for a discussion on Audio / Ultrasonic field recorders.

**Comment by Svardsten53** [comment]
Interesting thread! Before I found BirdNET-Pi, I spent some time on my own bird song recordings and analyzes with BirdNET-Lite in batch mode. Some of you may have seen what I wrote earlier: https://svardsten.eu/ljudanalys/nocmig/ It is written in Swedish, but Google Translate in the upper right corner is your friend. 
I have ordered an AudioMoth with weather proof case and estimated delivery is in February. An exciting little thing. More information is available here: https://www.openacousticdevices.info/

**Comment by DD4WH** [comment]
This is a DIY audio recorder:
https://github.com/WMXZ-EU/microSoundRecorder

Have a look at the WIKI, where there is a lot of information on programming, power demands, triggered recording, time scheduled recording, audio quality and a list of links and resources:
https://github.com/WMXZ-EU/microSoundRecorder/wiki

I have about 25 **microSoundRecorder** running here and I also own 4 AudioMoth (old and new version). If you can live with the **low audio quality of the AudioMoth**, its fine. If you want to build a recorder by yourself, customize your hardware and need higher audio quality and can live with a bit higher power demand, the microSoundRecorder is very nice. If money is no problem, you can buy the **excellent recorders by Wildlife Acoustics**, which combine both: long field duration and higher audio quality.

**Comment by oliverk71** [comment]
AudioMoth seem to be interesting. Is the built-in microphone capable of recording ultrasonic sounds, too? I guess not, but it would be so nice. Ultrasonic microphones are pretty expensive. There is another pi project, the bat pi project: http://www.bat-pi.eu/ I skipped it, because I did not want to spend so much money into an ultrasonic mic.

The specifications say: "Capable of recording at sample rates up to 384kHz", but I feel this is only valid for the board, not the microphone. But IF the AudioMoth including the built-in microphone is capable of recording up to 384 kHz, that would be really nice! 

Maybe someone knows? 

Another issue may be the low audio quality. I can't tell if it is sufficient for applications like birdnet-pi or bat-pi.

**Reply by DD4WH** [reply]
Divide the sample rate by two and you have the upper limit of frequency, so a device with 384ksps sample rate can record theoretically up to 192kHz. The AudioMoth mic has some ultrasonic response. My experience is that it is not suited for bat research due to the low sensitivity of the mic in the higher ultrasonics and because of the high noise level of the mic and the preamp (one stage of which is inside the mic that is used --> Knowles SPM0408LE5H (it has rather high selfnoise) and one stage inside the microcontroller used --> PGA == opamps in the EFM32 Gecko: they are both quite noisy). But your mileage may vary.

Alternative:
For bats, take a look at the **_TeensyRecorder by Jean-Do. Vrignault_**. I use many of those devices every summer and they can be called semiprofessional quality capable of recording bats with up to 500ksps sample rate and a power autonomy of up to two weeks depending on your usage cycle. They also have two superb microphones to choose from and the developer is super-motivated and kind. However, you have to build it by yourself or join one of the workshops where it is built. Highly recommended and only a little bit more expensive:
https://framagit.org/PiBatRecorderPojects/TeensyRecorders

**Reply by DD4WH** [reply]
Microphones with ultrasonic response are NOT always expensive, most electret capsule have some ultrasonic response as well as most MEMS mics.
Microphone capsules with good tested ultrasonic response:
* FG23329 electret about 35,- €, many professional bat recorders use this capsule [easy to solder]
* ICS40730 MEMS, about 3,- €, excellent sensitivity, low self noise and very good response up to 160kHz [without PCB and hot air solder station, this MEMS is quite a challenge to solder :-), but on the homepage of the TeensyRecorder you will find Email adresses where you can ask if you want to buy a presoldered PCB]

**Comment by timsterc** [comment]
If anyone has questions, I own and have used:

- Audiomoth
- Swift Recorder (Cornell)
- Song Meter Micro (Wildlife Acoustics)

I've not done any ultrasonic tests with any of them, my primary use has been as (bird) drop-off recorders, which are then run through a BirdNET-Lite script on my Macbook, with results pushed to BirdWeather for visualization.

**Reply by nilspupils** [reply]
What a great group this is!

**Reply by mcguirepr89** [reply]
I AGREE!
